src/flamenco/runtime/tests/fd_exec_sol_compat.c

```diff
@@ -82,17 +57,25 @@ sol_compat_wksp_init( void ) {
   smem = malloc( smax );  /* 1 GiB */
   assert( smem );
 
-  features.struct_size           = sizeof(sol_compat_features_t);
-  features.cleaned_up_features    = cleaned_up_features;
-  features.cleaned_up_feature_cnt = sizeof(cleaned_up_features)/sizeof(ulong);
-  features.supported_features    = supported_features;
-  features.supported_feature_cnt = sizeof(supported_features)/sizeof(ulong);
+  features.struct_size         = sizeof(sol_compat_features_t);
+  features.cleaned_up_features = malloc( 300 * sizeof(ulong) );
+  features.supported_features  = malloc( 300 * sizeof(ulong) );
+
+  for( const fd_feature_id_t * current_feature = fd_feature_iter_init(); !fd_feature_iter_done( current_feature ); current_feature = fd_feature_iter_next( current_feature ) ) {
```

Comment:
We should skip if the feature is reverted. Need to change the generator script to add reverted to the struct.